### PR TITLE
Create 2019-07-11--core-implementations-release-process.md

### PR DESCRIPTION
**Pull request by momack2**
@Stebalien @daviddias @alanshaw FYI

### CODE REVIEW

File: meeting-notes/2019/Q3/2019-07-11--core-implementations-release-process.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+
+* Action Items (AIs):
+   * Steven - Iterate on release process PR to the PR (https://github.com/ipfs/go-ipfs/pull/6482/files)
+   * David - Make a PR to go-libp2p and talk through it (meeting with Raul ++)
+      * https://github.com/libp2p/go-libp2p/issues/628
+   * Molly - comment on the release PR with these notes
+   * Steven - make a release issue w/ features + checklist for this patch release (0.4.22)
+   * Steven - update contributing guidelines for how describing features as PRs go along with text for release notes
+   * Alan and Steven - meet and record walkthroughs of the release process checklist
```
Comment (daviddias) [inline meeting-notes/2019/Q3/2019-07-11--core-implementations-release-process.md]: @alanshaw not sure if you were still on the call when we went through this. ^^ would be really cool

Comment (alanshaw) [reply meeting-notes/2019/Q3/2019-07-11--core-implementations-release-process.md]: I don't think I was - the current release checklist or the proposed one? Recording would be to explain to the community or to educate others who might be a DRI for a release in the future?

Comment (daviddias) [reply meeting-notes/2019/Q3/2019-07-11--core-implementations-release-process.md]: More of the second but the first would also happen as a consequence.

Comment (momack2) [reply meeting-notes/2019/Q3/2019-07-11--core-implementations-release-process.md]: @Stebalien - has this happened yet? Can you schedule/drive if not?

**Pull request merged**